srcpkgs/p7zip/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'p7zip'
 pkgname=p7zip
-version=16.02
-revision=5
-wrksrc="${pkgname}_${version}"
+version=17.03
+revision=1
 hostmakedepends="yasm nasm"
 short_desc="Command-line version of the 7zip compressed file archive"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="biopsin <biopsin@teknik.io>"
 license="LGPL-2.1-or-later"
 homepage="http://p7zip.sourceforge.net/"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}_${version}_src_all.tar.bz2"
-checksum=5eb20ac0e2944f6cb9c2d51dd6c4518941c185347d4089ea89087ffdd6e2341f
+changelog="https://github.com/jinfeihan57/p7zip/blob/master/ChangeLog"
+distfiles="https://github.com/jinfeihan57/p7zip/archive/v${version}.tar.gz"
```

Comment:
What makes this fork authoritative? This is doubly relevant because the homepage is still p7zip.sourceforge.net, but that site does not appear to direct people to the new fork.

Reply:
For the record, the fork itself doesn't try to claim to be an authoritative one.

Reply:
Yes, which makes me wonder why this is the preferred fork. Arch uses it, but it would be nice to see the original sanction this.

Reply:
I've forwarded the question to dev, awaiting response..

Reply:
> The p7zip hasn't been updated since 2016 and I can't reach the author of the p7zip.
> Then I contacted Igor Pavlov, the author of the 7zip, who was unable to reach the author of the p7zip.
> So I discussed with the author of 7zip to re-update p7zip. This branch really didn't get authorization from the original author of p7zip (because he can't be reached). Anyone who can contact the author of p7zip please let me know. I'll talk to him about the authorization.